packages/otelbin/src/components/monaco-editor/ValidationErrorConsole.tsx

```diff
@@ -42,58 +51,123 @@ export default function ValidationErrorConsole({ errors, font }: { errors?: IErr
 		}
 	}, [errorCount, warningsCount]);
 
+	const onHeightChange = useCallback((newHeight: number) => {
```

Comment:
**Question:** Why do we need this `useCallback` fn? As far as I can see, we could just use `setHeight` within the `useEffect` call below.

Reply:
fixed and pushed. @bripkens